src/Ryujinx/Input/GTK3/GTK3Mouse.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Ryujinx.Input.GTK3
 {
-    public class GTK3Mouse : IMouse
+    public class Gtk3Mouse : IMouse
```

Comment:
```suggestion
    public class GTK3Mouse : IMouse
```

Same as other comment